Packages/Application/Neos.NeosIo.Event/Resources/Private/TypoScript/NodeTypes/Sponsor.Opportunity.ts2

```diff
@@ -0,0 +1,18 @@
+/**
+* NodeTypes around Sponsor Opportunities
+*/
+
+prototype(Neos.NeosIo.Event:Sponsor.Opportunity.Document) < prototype(Neos.NeosIo.Event:DefaultPage) {
+	body {
+		content {
+			main = Neos.NeosIo.Event:Sponsor.Opportunity
+		}
+	}
+}
+
+prototype(Neos.NeosIo.Event:Sponsor.Opportunity) < prototype(TYPO3.Neos:Content) {
+	main = ContentCollection {
```

Comment:
Another nitpick: I think, `ContentCollection` should be addresses by it's fully qualified namespace, like with `Content` above - just for consistency